mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/tabs/index.tsx

```diff
@@ -17,7 +17,7 @@ export function Tabs(): JSX.Element {
     hasEmails: s(editorStoreName).automationHasStep('mailpoet:send-email'),
   }));
   const pageParams = useMemo(
-    () => new URLSearchParams(location.search),
+    () => new URLSearchParams(location.search as string),
```

Comment:
This looks like the type was completely lost. My guess is that it's because `history` is in version `4`, while `@types/history` was updated to version 5. Also, it seems that `@types/history` is deprecated now: https://www.npmjs.com/package/@types/history